package.json

```diff
@@ -8,7 +8,7 @@
     "node": ">=4.0.0"
   },
   "repository": "googleapis/nodejs-logging-bunyan",
-  "main": "./src/index.js",
+  "main": "./build/src/index.js",
```

Comment:
Don't forget to update the `files` section to include `build/src`

Reply:
Added in files section now, PTAL if this is the one you meant.

Reply:
Perfect!